### DISCUSSION TITLE
Resize columns without eating into adjacent column?

### DISCUSSION THREAD

**Discussion by zehawki** [table-library/react-table-library#46] [Q&A] [answered]
Is there a way to do an Excel style column resizing, where all the subsequent columns move, keeping their individual widths the same?

Currently resizing a column eats into the adjacent column, thus (kind of) defeating the purpose.

**Comment by rwieruch** [comment, marked as answer]
Good idea. At the moment, resizing always keeps the absolute size the same. However, for a horizontal scrollable table, this feature would make sense. Not possible in its current form, but will definitely think about it.

**Reply by iakovalgaev** [reply]
Yes, I am also really looking forward to this opportunity.
